packages/electrode-archetype-react-app/support/optimize-modules-for-production.js

```diff
@@ -5,6 +5,7 @@ const assert = require("assert");
 const fs = require("fs");
```

Comment:
this file is for supporting the app at run time, and for that we want to use `console.log` only.

We want the logger to be used only during development/build time.

That also mean we don't want the logger (winston/bunyan) to be a prod dep for the app.